### PR TITLE
fix: [PIE-8130]: checkbox selection not working in compare executions

### PR DESCRIPTION
**Pull request by udayvunnam**
### Summary

- checkbox selection isn't working in compare executions due to preventDefault even in the parent
- some UX fixes

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/70-pipeline/components/ExecutionCompareYaml/ExecutionCompareYamlHeader.tsx [hunk 1]
```diff
@@ -20,9 +21,9 @@ export function ExecutionCompareYamlHeader(): ReactElement {
 
   return (
     <>
-      <Page.SubHeader>
```
Comment (udayvunnam) [inline src/modules/70-pipeline/components/ExecutionCompareYaml/ExecutionCompareYamlHeader.tsx:23]: This is already inside a Page.SubHeader

File: src/modules/70-pipeline/pages/execution-list/ExecutionListTable/ExecutionListCells.tsx [hunk 2]
```diff
@@ -143,7 +142,7 @@ export const RowSelectCell: CellType = ({ row }) => {
   }
 
   return (
-    <div ref={checkboxRef} className={css.checkbox} onClick={killEvent}>
```
Comment (udayvunnam) [inline src/modules/70-pipeline/pages/execution-list/ExecutionListTable/ExecutionListCells.tsx:146]: unnecessary ref

File: src/modules/72-freeze-windows/components/FreezeWindowList/ToggleAllSelection.tsx [hunk 3]
```diff
@@ -16,7 +15,7 @@ export const ToggleAllSelection: FC<{ data: FreezeSummaryResponse[]; canEdit: bo
   const { toggleAllSelect, selectedItems } = useFreezeWindowListContext()
 
   return (
-    <div className={css.checkbox} onClick={killEvent}>
```
Comment (udayvunnam) [inline src/modules/72-freeze-windows/components/FreezeWindowList/ToggleAllSelection.tsx:19]: not required here

**Review by ishantmehtaharness — APPROVED** [review]
Looks Good Uday

**Pull request merged**